src/components/NavBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { NavLink } from 'react-router-dom';
+
+export const Navbar = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => `navbar-item ${isActive ? 'has-background-grey-lighter' : ''}`}
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            to="/people"
+            className={({ isActive }) => `navbar-item ${isActive ? 'has-background-grey-lighter' : ''}`}
```

Comment:
use classnames

And here is an example of how to do it without an empty class. But use classnames))
```suggestion
            className={({ isActive }) => `navbar-item ${isActive ?? 'has-background-grey-lighter'}`}
```